modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -330,6 +330,10 @@ private static IgniteTuple3<List<Lease>, Boolean, Boolean> replaceProlongableLea
         ClusterNode primaryCandidate = null;
 
         for (Assignment assignment : assignments) {
```

Comment:
Previously I suggested to rename variables like `deserFunction`, but there is one left: in parameters of `AssignmentsTracker#updateGroupAssignments`. Would you rename it please?